cogs/music.py

```
@@ -0,0 +1,581 @@
+            player = MusicPlayer(ctx)
+            self.players[ctx.guild.id] = player
+            
+        return player
+    
+    async def is_valid_url(self, url):
+        """Check if URL is valid and accessible."""
+        try:
+            session = await self.get_session()
+            async with session.head(url, timeout=5) as resp:
+                return resp.status == 200
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)